### PR TITLE
Update build instructions for adapter

### PR DESCRIPTION
**Pull request by BenjaminRodenberg**
I'm trying to build the adapter now. Currently I do not succeed, but I will step-by-step update this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,14 +3,9 @@
...
-  * The variable SOURCES needs one main udf file and the correspoding .c files:
-    + fsi_udf.c: This is the main ANSYS readable file used by Fluent to call functions during each iteration
-                 for FSI simulations. This ANSYS UDF file needs fsi.c and fsi.h for execution
-  * Adapt `lnamd64/2d_host/user.udf` line 3 `FLUENT_INC = ...` to the fluent installation
-    folder which will be of the type `./ansys_inc/v195/fluent`
-  * Repeat the above steps for `lnamd64/2d_node` folder in the exact same manner
-  * Adapt the path of the python library in `src/makefile` line 19.  
+  * Adapt `lnamd64/2ddp_host/user.udf` line 3 `FLUENT_INC = ...` to the fluent installation (for example: `/home/myusername/ansys_inc/v195/fluent`, if you installed Fluent, Ansys 2019 R3, into your home directory).
+  * Repeat the step above for `lnamd64/2ddp_node` folder in the exact same manner.
+  * Adapt the path of the python library in `src/makefile` line 19. (**TODO** Where is the makefile? How do I get the makefile?)
```
Comment (BenjaminRodenberg) [inline README.md]: I Cannot find a makefile.

File: README.md [hunk 2]
```diff
@@ -3,14 +3,9 @@
 *Updated and maintained done by Ishaan Desai*
 
 ## 1. How to build the Fluent-preCICE adapter:
-  * Adapt `lnamd64/2ddp_host/user.udf` line 1 "CSOURCES=...": There are several main udf files
-  * The variable SOURCES needs one main udf file and the correspoding .c files:
-    + fsi_udf.c: This is the main ANSYS readable file used by Fluent to call functions during each iteration
-                 for FSI simulations. This ANSYS UDF file needs fsi.c and fsi.h for execution
```
Comment (BenjaminRodenberg) [inline README.md:9]: I do not have to change anything here. Therefore, I removed this line. Or is there anything that has to be added?

File: README.md (hunk 1 above)
Comment (IshaanDesai) [reply README.md]: I am still figuring out a way to provide a recipe for a ANSYS user to construct the `makefile` from a given file in the ANSYS code. Once this recipe is fixed the `makefile` will be available for use.

File: README.md (hunk 2 above)
Comment (IshaanDesai) [reply README.md:9]: These lines are there as an information to the user as to where the adapter source code is inserted and seen by ANSYS. if in the future there are more files (for e.g. `cht.c` for a CHT application), then the `"CSOURCES=..."` will need to be modified accordingly. 
I would suggest to keep these lines for the moment.

**Review by IshaanDesai — CHANGES_REQUESTED** [review]
More discussion necessary before finalizing build instructions

**Comment by BenjaminRodenberg** [comment]
Let's try to close or merge this PR. I don't know, if the changes or instructions are useful at the current state. But I think we will have to change a lot anyway and putting too much effort in this PR is probably a waste of time. Any opinion here @mtree or @IshaanDesai?

**Review by IshaanDesai — CHANGES_REQUESTED** [review]
I think this PR is too old and has potentially diverged from the perspective of last weeks discussion

**Comment by BenjaminRodenberg** [comment]
> I think this PR is too old and has potentially diverged from the perspective of last weeks discussion

Ok. Then I will close this PR, because I agree that updating this PR is not leading into the right direction. Please don't forget to move potentially useful information from here to #24.

**Pull request closed without merging**